RevenueCat/Example/PurchasesListener.cs

```diff
@@ -1,23 +1,51 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 using UnityEngine.Events;
 using UnityEngine.UI;
 
 public class PurchasesListener : Purchases.UpdatedCustomerInfoListener
 {
-
     public RectTransform parentPanel;
     public GameObject buttonPrefab;
-    public Text customerInfoLabel;
+    public Text infoLabel;
+
+    private bool simulatesAskToBuyInSandbox;
+
+    private int minYOffsetForButtons = 40; // values lower than these don't work great with devices
+    // with safe areas on iOS
+
+    private int minXOffsetForButtons = 20;
+
+    private int xPaddingForButtons = 10;
+    private int yPaddingForButtons = 5;
+
+    private int maxButtonsPerRow = 2;
+    private int currentButtons = 0;
 
     // Use this for initialization
     private void Start()
     {
-        CreateButton("Restore Purchases", RestoreClicked, 100);
-
-        CreateButton("Switch Username", SwitchUser, 200);
-
-        CreateButton("Do Other Stuff", DoOtherStuff, 300);
+        CreateButton("Get Customer Info", GetCustomerInfo);
+        CreateButton("Get Offerings", GetOfferings);
+        CreateButton("Sync Purchases", SyncPurchases);
+        CreateButton("Restore Purchases", RestorePurchases);
+        CreateButton("Can Make Payments", CanMakePayments);
+        CreateButton("Set Subs Attributes", SetSubscriberAttributes);
+        CreateButton("Log in as \"test\"", LogInAsTest);
+        CreateButton("Log in as random id", LogInAsRandomId);
+        CreateButton("Log out", LogOut);
+        CreateButton("Check Intro Eligibility", CheckIntroEligibility);
+        CreateButton("Get Promo Offer", GetPromotionalOffers);
+        CreateButton("Buy package w/discount", BuyFirstPackageWithDiscount);
+        CreateButton("Buy product w/discount", BuyFirstProductWithDiscount);
+        CreateButton("Code redemption sheet", PresentCodeRedemptionSheet);
+        CreateButton("Invalidate customer info cache", InvalidateCustomerInfoCache);
+        CreateButton("Get all products", GetAllProducts);
+        CreateButton("Toggle simulatesAskToBuyInSandbox", ToggleSimulatesAskToBuyInSandbox);
+        CreateButton("Is Anonymous", IsAnonymous);
+        CreateButton("Get AppUserId", GetAppUserId);

```

Comment:
look at all the things you can do now 🎉 

Also, to add something new, you just need to call `CreateButton`